fe/fe-core/src/main/java/com/starrocks/catalog/FunctionSet.java

```diff
@@ -751,6 +762,26 @@ public class FunctionSet {
     public static final Set<String> INDEX_ONLY_FUNCTIONS =
             ImmutableSet.<String>builder().add().add(NGRAM_SEARCH).add(NGRAM_SEARCH_CASE_INSENSITIVE).build();
 
+    // Unsupported functions for agg state combinator.
+    public static final Set<String> UNSUPPORTED_AGG_STATE_FUNCTIONS =
+            new ImmutableSortedSet.Builder<>(String.CASE_INSENSITIVE_ORDER)
+                    .add(COUNT) // Functions with pseudo types are not supported.
+                    .add(COUNT_IF)
+                    .add(FLAT_JSON_META) // Internal functions are not supported.
+                    .add(HLL_RAW)
```

Comment:
COUNT, COUNT_IF is used frequently, so we need to support it.

NDV/APPROX_COUNT_DISTINCT(c) = HLL_CARDINALITY(HLL_RAW(c));
HLL_CARDINALITY(HLL_UNION(HLL_HASH(c))) = HLL_CARDINALITY(HLL_RAW(cast(c as STRING)))

so, in fact, we only need to implement two hll agg functions,  hll_raw和hll_union, other functions derived agg functions need not be re-implemented,  just use other functions to convert input/output parameters.

Reply:
1. `count/array_agg` have been supported
2. `hll_raw` is not supported because it uses `hll` as input which has been supported in `hll_union` in aggregate table.
3. you are right, it's only true that we only need to implement some agg functions' state(eg: array_agg) to recompute other functions state which may be like implementions in the streaming system. But that's a huge work to recompute agg functions by the limited agg function state and we can discuss this later.